ansible/roles/schulcloud-server-core/templates/deployment.yml.j2

```diff
@@ -49,6 +49,9 @@ spec:
         - containerPort: 3030
           name: api
           protocol: TCP
+        - containerPort: 4030
+          name: api-admin
+          protocol: TCP
```

Comment:
The server aka api don't start the admin api so the port cann be removed
```suggestion
```

Reply:
done